hicommonwealth/commonwealth
# Fix scrollbars display on Windows/Linux

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8656

## Description of Changes
- Fixed scrollbars display on non-macOS devices
- Fixed scrollbars hidden behind breadcrumbs component

## "How We Fixed It"
N/A

## Test Plan
- Visit the site on non-macOS devices and verify scrollbars are similar to those displayed on macOS
- Resize screen on different pages and verify scrollbars are not hidden behind breadcrumbs component

## Deployment Plan
N/A

## Other Considerations
N/A
